### ISSUE TITLE
Build failed: incompatible with new updates of tomochain

### ISSUE THREAD

**Issue by thanhnguyennguyen** [thanhnguyennguyen/tomox-bot#9]
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [thanhnguyennguyen](https://github.com/thanhnguyennguyen)

**Comment by thanhnguyennguyen** [comment]
```bash
package github.com/tomochain/tomochain/tomox/tomox_state: cannot find package "github.com/tomochain/tomochain/tomox/tomox_state" in any of:
	/usr/local/go/src/github.com/tomochain/tomochain/tomox/tomox_state (from $GOROOT)
	/go/src/github.com/tomochain/tomochain/tomox/tomox_state (from $GOPATH)
	/go/src/github.com/thanhnguyennguyen/tomox-bot/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
	/go/src/github.com/thanhnguyennguyen/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
	/go/src/github.com/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
Compiling for linux/amd64...
package github.com/tomochain/tomochain/tomox/tomox_state: cannot find package "github.com/tomochain/tomochain/tomox/tomox_state" in any of:
	/usr/local/go/src/github.com/tomochain/tomochain/tomox/tomox_state (from $GOROOT)
	/go/src/github.com/tomochain/tomochain/tomox/tomox_state (from $GOPATH)
	/go/src/github.com/thanhnguyennguyen/tomox-bot/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
	/go/src/github.com/thanhnguyennguyen/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
	/go/src/github.com/Godeps/_workspace/src/github.com/tomochain/tomochain/tomox/tomox_state
2020/03/14 13:37:57 Failed to cross compile package: exit status 1.
It took 2 minutes

```
`tomox_state`
This package has been renamed to `tradingstate`

**Comment by thanhnguyennguyen** [comment]
fixed in 41df44bd6d9f490076b345534cb5466dd1e22f96
